disnake/components.py

```diff
@@ -39,6 +40,8 @@
     cast,
 )
 
+from typing_extensions import TypeAlias
```

Comment:
should be imported in the `if TYPE_CHECKING` block, not imported at runtime (see rtd build)